### ISSUE TITLE
units | Malakar is very weak for level 2/3/4; give him some traits

### ISSUE THREAD

**Issue by knyghtmare** [cooljeanius/Flight_Freedom#11]
May I suggest some appropriate drake-like stats?

I think I have an up-to-date Drake Legend/Lord as well.

**Comment by cooljeanius** [comment]
kinda ties in with #2 IMO

**Comment by knyghtmare** [comment]
> kinda ties in with #2 IMO

No. It doesn't. I wish to update Malakar to the current Drake style. He is very archaic looking. Like 1.4 type.

**Comment by knyghtmare** [comment]
also, level 3 advances to level 3? What the...
I need to adjust things

**Comment by cooljeanius** [comment]
> > kinda ties in with #2 IMO
> 
> No. It doesn't. I wish to update Malakar to the current Drake style. He is very archaic looking. Like 1.4 type.

I think that's part of his charm

**Comment by knyghtmare** [comment]
> > > kinda ties in with #2 IMO
> > 
> > 
> > No. It doesn't. I wish to update Malakar to the current Drake style. He is very archaic looking. Like 1.4 type.
> 
> I think that's part of his charm

Nah, I got a better Drake Lord set in my resources. Let me update Malakar to it. Maybe next week or so.

**Comment by cooljeanius** [comment]
> > > > kinda ties in with #2 IMO
> > > 
> > > 
> > > No. It doesn't. I wish to update Malakar to the current Drake style. He is very archaic looking. Like 1.4 type.
> > 
> > 
> > I think that's part of his charm
> 
> Nah, I got a better Drake Lord set in my resources. Let me update Malakar to it. Maybe next week or so.

No, his sprite is staying as-is. Sorry.

**Comment by cooljeanius** [comment]
Another reason for keeping the sprite as-is (besides the fact that I like it as-is): All the story images would have to be re-rendered if we changed it

**Comment by cooljeanius** [comment]
So, I was considering closing this given the progress I've been making on #2, but then I realized that Malakar doesn't have any traits, so I might give him some to make him stronger, as per this issue... which traits do you think would make sense?

**Comment by knyghtmare** [comment]
traits are irrelevant when the base unit looks so weak

**Comment by cooljeanius** [comment]
actually I guess I'll wait and see what the resolution to wesnoth/wesnoth#6712 is before deciding on traits

**Comment by knyghtmare** [comment]
> actually I guess I'll wait and see what the resolution to [wesnoth/wesnoth#6712](https://github.com/wesnoth/wesnoth/issues/6712) is before deciding on traits

that has very low chance of ever happening. Follow the argument. Hejnewar is against it and eval's arguments are lacking substance.

**Comment by cooljeanius** [comment]
Kinda related: wesnoth/wesnoth#7129. @Ashmyr WDYT?

**Comment by cooljeanius** [comment]
> also, level 3 advances to level 3? What the... I need to adjust things

This part is fixed as of 5.4.1, btw

**Comment by Ashmyr** [comment]
I don't remember playing this campaign so I can't give input.

**Comment by cooljeanius** [comment]
> I don't remember playing this campaign so I can't give input.

You can read up on it here: https://wiki.wesnoth.org/FlightToFreedom

**Comment by cooljeanius** [comment]
Tempted to close this after [c020f04](https://github.com/cooljeanius/Flight_Freedom/commit/c020f04364ffb86d47367f5cdd74210047a2196b), but I should probably test it first... 
(well, and I still want to give him traits, but still haven't decided on any yet...)

**Comment by cooljeanius** [comment]
> Tempted to close this after [c020f04](https://github.com/cooljeanius/Flight_Freedom/commit/c020f04364ffb86d47367f5cdd74210047a2196b), but I should probably test it first... (well, and I still want to give him traits, but still haven't decided on any yet...)

Actually I guess I could do like I did in DoT and copy over that macro from "A Group in a War" that would let the player choose his traits...

**Comment by cooljeanius** [comment]
> > Tempted to close this after [c020f04](https://github.com/cooljeanius/Flight_Freedom/commit/c020f04364ffb86d47367f5cdd74210047a2196b), but I should probably test it first... (well, and I still want to give him traits, but still haven't decided on any yet...)
> 
> Actually I guess I could do like I did in DoT and copy over that macro from "A Group in a War" that would let the player choose his traits...

The problem with that, though, is that the first scenario starts you off "*in media res*", so a "choose your traits" type dialog wouldn't really feel appropriate during it... thinking of going with @ForestDragon-wesnoth's suggestion from Discord and just inventing a custom trait instead...

**Comment by cooljeanius** [comment]
Hey @MadMax3947 WDYT?

**Comment by cooljeanius** [comment]
[5.4.2](https://github.com/cooljeanius/Flight_Freedom/releases/tag/5.4.2) improves his L4 stats a bit, but I'm leaving this open until I come up with traits for him

**Comment by MadMax3947** [comment]
https://github.com/MadMax3947/Flight_Freedom/commit/ea86b1eb6f8fc15e5a798a9289a8679cee08c490 gives L3/L4 Malakar a bit of a buff, particularly against enemies with fire resistance.

**Comment by cooljeanius** [comment]
> [MadMax3947@ea86b1e](https://github.com/MadMax3947/Flight_Freedom/commit/ea86b1eb6f8fc15e5a798a9289a8679cee08c490) gives L3/L4 Malakar a bit of a buff, particularly against enemies with fire resistance.

I guess that's probably enough; I'll close this once #47 is merged, since that PR includes that commit...

**Comment by cooljeanius** [comment]
Closed with a9f978f

**Comment by cooljeanius** [comment]
Hey @knyghtmare please take a look at PR #51
